docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md

```diff
@@ -392,7 +392,7 @@ Notice that it takes a few seconds to display the counts. During that time, the
 
 ## Replace method GetURLContentsAsync with a .NET method
 
-1. .NET Framework 4.5 and later versions provide many async methods that you can use. One of them, the <xref:System.Net.Http.HttpClient> method <xref:System.Net.Http.HttpClient.GetByteArrayAsync%28System.String%29>, does just what you need for this walkthrough. You can use it instead of the `GetURLContentsAsync` method that you created in an earlier procedure.
+1. .NET Framework 4.5 and higher versions provide many async methods that you can use. One of them, the <xref:System.Net.Http.HttpClient> method <xref:System.Net.Http.HttpClient.GetByteArrayAsync%28System.String%29>, does just what you need for this walkthrough. You can use it instead of the `GetURLContentsAsync` method that you created in an earlier procedure.
```

Comment:
```suggestion
1. .NET Framework 4.5 and later versions provide many async methods that you can use. One of them, the <xref:System.Net.Http.HttpClient> method <xref:System.Net.Http.HttpClient.GetByteArrayAsync%28System.String%29>, does just what you need for this walkthrough. You can use it instead of the `GetURLContentsAsync` method that you created in an earlier procedure.
```